deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/CommonItemConfig.java

```diff
@@ -48,6 +48,12 @@ public class CommonItemConfig {
     @ConfigItem(name = "additional-api-type-annotations")
     public Optional<String> additionalApiTypeAnnotations;
 
+    /**
+     * Add custom/additional HTTP Headers or other args to every request
+     */
+    @ConfigItem(name = "additional-request-args")
+    public Optional<String> additionalRequestArgs;
```

Comment:
IHO, I think that the property could be `additional-request-headers` or something like it. WYT @hbelmiro?

Reply:
@ru4ert I think your not limited to just headers. See my Test

Reply:
Makes sense!